src/creature.cpp

```diff
@@ -1327,7 +1322,7 @@ int64_t Creature::getStepDuration() const
 	double duration = std::floor(1000 * groundSpeed / calculatedStepSpeed);
 	int64_t stepDuration = std::ceil(duration / 50) * 50;
 
-	const Monster* monster = getMonster();
+	auto monster = this->getMonster();
```

Comment:
Could this be `const auto`
```suggestion
	const auto monster = this->getMonster();
```